src/app/api/organizations/route.ts

```diff
@@ -0,0 +1,24 @@
+import { createOrganization } from '@/server/actions/Organizations';
+import { NextRequest, NextResponse } from 'next/server';
+
+// @route Post /api/organizations/ - Creates an organization
+
+export async function POST(request: NextRequest) {
+  try {
+    const req = await request.json();
+    const res = await createOrganization(req);
+
+    return NextResponse.json({ id: res }, { status: 201 });
+  } catch (error) {
+    if ((error = 11000)) {
```

Comment:
I like this pattern! How'd you find this code? Manual testing? I've looked around for some error code reference but never could find one.

Reply:
I was surfing around the internet, and found that this worked.

Reply:
was that just for this response code? If there's some central reference list, could you send it, that'd be great to have!